Sources/LiveViewNative/Views/Controls and Indicators/Value Inputs/Slider.swift

```diff
@@ -12,14 +12,14 @@ import SwiftUI
 /// By default, sliders choose values in the range 0-1.
 ///
 /// ```html
-/// <Slider value-binding="progress" />
+/// <Slider value="progress" />
 /// ```
 ///
 /// Use ``lowerBound`` and ``upperBound`` to specify the range of possible values.
 ///
 /// ```html
 /// <Slider
-///     value-binding="progress"
+///     value="progress"
 ///     lower-bound={-1}
 ///     upper-bound={2}
```

Comment:
why can't use use the `in` from the original SwiftUI view and pass a Range?

```
<Slider
  value="progress"
  in={-1..2}
/>
``

Reply:
This is not possible because `Range` doesn't implement `Phoenix.HTML.Safe` so it can't be encoded as an attribute.

Reply:
not supported *yet*